app/views/pets/index.html.erb

```diff
@@ -1,7 +1,7 @@
 <h2>All Pets</h2>
   <%@pets.each do |pet|%>
   <img src="<%=  pet.image %>" alt="<%= pet.image %>" style="width:200px">
-  <h4>Name: <%= pet.name %></h4>
+  <h4><a href="/pets/<%= pet.id %>">Name: <%= pet.name %></a></h4>
```

Comment:
Adds a link to every pet name shown on the page that leads to that pet's `/pets/:id` page.